traefikee/templates/_helpers.tpl

```diff
@@ -103,3 +103,17 @@ Generates or load registry token.
   {{- end }}
 {{- printf "%s" $tokenStr | nospace | b64enc }}
 {{- end }}
+
+{{/*
+Set imageVersion from Chart
+*/}}
+{{- define "imageVersion" -}}
+{{ (split "@" (default $.Chart.AppVersion $.Values.image.tag))._0 }}
```

Comment:
You might want to additionally handle images with `-ubi` or `-rootless` so you might need an extra split :-)